Upstream-Status: Pending

12/03/2010

add explicit rules for header files generated by lex and yacc,
otherwise the build of lib/route/pktloc.c may fail in a parallel
environment.

Signed-off-by: Qing He <qing.he@intel.com>

12/06/2010

the dependency rule should really read pktloc.lo instead of
pktloc.$(OBJEXT), since it's in a libtool setup.

Signed-off-by: Qing He <qing.he@intel.com>

diff --git a/lib/Makefile.am b/lib/Makefile.am
index 92a916e..e8b8ef3 100644
--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -35,6 +35,10 @@ route/pktloc_grammar.c: route/pktloc_grammar.l
 route/pktloc_syntax.c: route/pktloc_syntax.y
 	$(YACC) -d $(YFLAGS) -o $@ $^
 
+route/pktloc.lo: route/pktloc_syntax.h route/pktloc_grammar.h
+route/pktloc_syntax.h: route/pktloc_syntax.c
+route/pktloc_grammar.h: route/pktloc_grammar.c
+
 libnl_route_la_LDFLAGS = -version-info 2:0:0
 libnl_route_la_LIBADD  = libnl.la
 libnl_route_la_SOURCES = \
